tests/importer/onnx/basic/test_split.py

```diff
@@ -0,0 +1,153 @@
+    for i in range(output_size):
+        output_name = 'output_{0}'.format(i)
+        output_names.append(copy.deepcopy(output_name))
+        out_shape[dim] = split[i] if split is not None else in_shape[dim] // output_size
+        output = helper.make_tensor_value_info(output_name, TensorProto.FLOAT, out_shape)
+        outputs.append(copy.deepcopy(output))
+
+    if axis is not None:
+        attributes_dict['axis'] = axis
+
+    # opset 1/2/11
+    if op_version == 1 or op_version == 2 or op_version == 11:
+        if split is not None:
+            attributes_dict['split'] = split
+    else:
+        # opset 13
+        if split is not None:
+            split_tensor = helper.make_tensor(
+                'split',
+                TensorProto.INT64,
+                dims=[len(split)],
+                vals=split
+            )
+
+            if value_format == 'initializer':
+                initializers.append(split_tensor)
+            else:
+                const_node = helper.make_node(
+                    'Constant',
+                    inputs=[],
+                    outputs=['split'],
+                    value=split_tensor
+                )
+                nodes.append(const_node)
+
+            input_names.append('split')
+
+    split_node = onnx.helper.make_node(
+        'Split',
+        inputs=input_names,
+        outputs=output_names,
+        **attributes_dict
+    )
+    nodes.append(split_node)
+
+    graph_def = helper.make_graph(
+        nodes,
+        'test-model',
+        inputs,
+        outputs,
+        initializer=initializers
+    )
+
+    op = onnx.OperatorSetIdProto()
+    op.version = op_version
+    model_def = helper.make_model(graph_def, producer_name='kendryte', opset_imports=[op])
+
+    return model_def
+
+
+in_shapes = [
+    [4, 8, 8]
+]
+
+axes = [
+    None,
+    -3
+]
+
+splits = [
+    None,
+    [2, 2],
+    [1, 1, 2],
+    [1, 2, 1],
+    [2, 1, 1],
+    [1, 1, 1, 1]
+]
+
+output_sizes = [
+    2,
+    3,
+    4
+]
+
+op_versions_and_value_formats = [
+    # opset 1 is not supported by onnx runtime yet.
+    # [1, 'attribute'],
+    [2, 'attribute'],
+    [11, 'attribute'],
+    [13, 'initializer'],
+    [13, 'constant']
+]
+
+
+@pytest.mark.parametrize('in_shape', in_shapes)
+@pytest.mark.parametrize('axis', axes)
+@pytest.mark.parametrize('split', splits)
+@pytest.mark.parametrize('output_size', output_sizes)
+@pytest.mark.parametrize('op_versions_and_value_format', op_versions_and_value_formats)
+def test_split(in_shape, axis, split, output_size, op_versions_and_value_format, request):
+    op_version, value_format = op_versions_and_value_format
+    dim = axis if axis is not None else 0
+    if (split is None and in_shape[dim] % output_size == 0) or (split is not None and len(split) == output_size):
+        model_def = _make_module(in_shape, axis, split, output_size, op_version, value_format)
+
+        runner = OnnxTestRunner(request.node.name, ['k510'])
```

Comment:
Why not other targets?

Reply:
Forgot to remove the test target. I'll enable all of targets.